mpa/modules/utils/mask_utils.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+
+def get_ignored_labels_per_batch(img_metas, num_classes):
+    ignored_labels = []
+    for _, meta in enumerate(img_metas):
+        valid_labels = torch.Tensor([1 for _ in range(num_classes)])
+        if 'ignored_labels' in meta and meta['ignored_labels']:
+            valid_labels[meta['ignored_labels']] = 0
+        ignored_labels.append(valid_labels)
+    return ignored_labels
```

Comment:
mask_utils.py <- this name could be misleading. Is this a utility function for mask?

Reply:
Is this function getting 'valid' labels per batch? Or 'ignored' labels per batch?
@harimkang What do you think? Name and contents are not consistent.

Reply:
It's getting 'valid' labels per batch. Let me rename it to  get_valid_labels_per_batch.

Reply:
> 
mask_utils.py was created to put common function named get_valid_labels_per_batch, which are called in custom fcn/ocr heads. What do you think of renaming it to seg_utils.py?

Reply:
I will modify the detection side to the same name ('ignored' -> 'valid') 
and the name of the function that gets the mask to 'get_valid_labels_per_batch'.

Reply:
I had modified it to 'valid' yesterday. But valid_labels name has been already using in mmseg. So I need to modify the name again. (We cannot name it to valid_labels or valid_mask because they have been using in mmseg.) 

What do you think of keeping using ignored_labels or ignored_mask?

Reply:
How about valid_label_mask? This seems more precisely correct for this code semantic. If you want to keep it as ignored_xxx, we need to flip 0/1 values.
valid = [0, 0, 1, 0] -> ignored = [1, 1, 0, 1]

Reply:
The valid_label_mask looks good, too.

Reply:
Looks good. Let me rename it to valid_label_mask. Thanks for your suggestion.